compiler/luci/pass/src/ModulePhase.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PHASE_H__
+#define __LUCI_PHASE_H__
+
+#include <luci/LuciPass.h>
```

Comment:
Not related with this PR... 
I didn't catch this filename in the previous review... 
If you agree, could we rename this (in another PR) to `ModulePass`?

Reply:
At first, I named it as `ModulePass` as you said.
However, I was worried about a case that both `run(luci::Module *m)` and `run(loco::Graph *g)` are needed.

For example, `ShapeInferencePass`, `TypeInferencePass`, `ShapeSigantureInferencePass` may be needed to be done not only for `module` passes but also for `graph` passes.
I thought that `ModulePass` may be confused to describe this situation and thus concluded as following names.

- `LuciPass` includes : `run(luci::Module *)` (Mandatory) / `run(loco::Graph *)` (Optional)
- `logo::Pass` includes : `run(loco::Graph *)` (Mandatory)

If you agree that `ModulePass` is better than `LuciPass`, I will gladly change the name :)

Reply:
@seanshpark As we discussed at offline, I will change it after this PR is merged! :)